src/packages/blits/src/utils/typeGuards.ts

```diff
@@ -0,0 +1,23 @@
+import type {Tone, States, JustifyContent, Align} from "../types/types";
+
+// type guards for Tone/States/JustifyContent
+export const ToneValues = ['neutral', 'inverse', 'brand'] as const;
+export function isValidTone(v: unknown): v is Tone {
+  return typeof v === 'string' && ToneValues.includes(v as Tone);
+}
+
+export const StatesValues = ['focus', 'unfocused', 'disabled'] as const;
```

Comment:
I think this is probably fine for now, but tones and modes(states) can be configured on a per-component basis. My tone and mode types are also more or less hardcoded 😆 but i just wanted to flag it

Reply:
Thanks Anthony I'll have a think about a better way to do this, I didn't realise about the per-component types